src/cr/cube/matrix/cubemeasure.py

```diff
@@ -161,6 +166,89 @@ def means(self):
         return self._means[:, 0, :, 0]
 
 
+# === STD DEV ===
+
+
+class _BaseCubeStdDev(_BaseCubeMeasure):
+    """Base class for stddev cube-measure variants."""
+
+    def __init__(self, dimensions, stddev):
+        super(_BaseCubeStdDev, self).__init__(dimensions)
+        self._stddev = stddev
+
+    @classmethod
+    def factory(cls, cube, dimensions, slice_idx):
+        """Return _BaseCubeStdDev subclass instance appropriate to `cube`.
+
+        Raises `ValueError` if the cube-result does not include a cube-stddev measure.
+        """
+        if cube.stddev is None:
+            raise ValueError("cube-result does not contain cube-stddev measure")
+        dimension_types = cube.dimension_types[-2:]
+        CubeSumsCls = (
+            _MrXMrCubeStdDev
+            if dimension_types == (DT.MR, DT.MR)
+            else _MrXCatCubeStdDev
+            if dimension_types[0] == DT.MR
+            else _CatXMrCubeStdDev
+            if dimension_types[1] == DT.MR
+            else _CatXCatCubeStdDev
+        )
+        return CubeSumsCls(
+            dimensions, cube.stddev[cls._slice_idx_expr(cube, slice_idx)]
+        )
+
+    @lazyproperty
+    def stddev(self):
```

Comment:
Do we want to use `_base_values` instead of `stddev` for consistency?

Reply:
actually each BaseCubeMeasure have it's measure name property and not base_values. maybe i'm not following you. Can you explain your concern?